src/porepy/compositional/materials.py

```diff
@@ -0,0 +1,476 @@
+            "lame_lambda": "Pa",
+            "maximum_elastic_fracture_opening": "m",
+            "normal_permeability": "m^2",
+            "permeability": "m^2",
+            "porosity": "-",
+            "residual_aperture": "m",
+            "shear_modulus": "Pa",
+            "skin_factor": "-",
+            "specific_heat_capacity": "J * kg^-1 * K^-1",
+            "specific_storage": "Pa^-1",
+            "temperature": "K",
+            "thermal_conductivity": "W * m^-1 * K^-1",
+            "thermal_expansion": "K^-1",
+            "well_radius": "m",
+            "open_state_tolerance": "-",
+            "contact_mechanics_scaling": "-",
+        }
+    )
+
+    biot_coefficient: number = 1
+
+    characteristic_contact_traction: number = 1
+
+    characteristic_displacement: number = 1
+
+    contact_mechanics_scaling: number = 1e-1
+    """Numerical method parameter
+
+    Safety scaling factor, making fractures softer than the matrix
+
+    """
+
+    density: number = 1
+
+    dilation_angle: number = 0
+
+    fracture_gap: number = 0
+
+    fracture_normal_stiffness: number = 1
+    """Intended use is in Barton-Bandis-type models for elastic fracture deformation."""
+
+    fracture_tangential_stiffness: number = -1.0
+    """
+    Note:
+        The current default value is -1.0, with the convention that negative
+        values correspond to a fracture that does not deform elastically in the
+        tangential direction.
+    """
+
+    friction_coefficient: number = 1
+
+    lame_lambda: number = 1
+    """Lame's first parameter"""
+
+    maximum_elastic_fracture_opening: number = 0
+    """Intended use is in Barton-Bandis-type models for elastic fracture deformation."""
+
+    normal_permeability: number = 1
+
+    open_state_tolerance: number = 1e-5
+    """Numerical method parameter.
+
+    Tolerance parameter for the tangential characteristic contact mechanics.
+
+    FIXME: Revisit the tolerance.
+
+    """
+
+    permeability: number = 1
+
+    porosity: number = 0.1
+
+    residual_aperture: number = 0.1
+
+    shear_modulus: number = 1
+
+    skin_factor: number = 0
+
+    specific_heat_capacity: number = 1
+
+    specific_storage: number = 1
+
+    temperature: number = 0
+
+    thermal_conductivity: number = 1
+
+    thermal_expansion: number = 0
+
+    well_radius: number = 0.1
+
+
+def load_fluid_constants(
+    names: list[str], package: str = "chemicals"
+) -> list[FluidConstants]:
+    """Creates a fluid species, if identifiable by ``name`` in ``package``.
+
+    Utility function to extract parameters for a fluid, like critical values.
+
+    Important:
+        The ``name`` is passed directly to the package. There is no guarantee if the
+        returned values are correct, of if the third-party package will work without
+        throwing errors.
+
+    Parameters:
+        names: A list of names or chemical formulae to look up the chemical species.
+        package: ``default='chemicals'``
```

Comment:
This function was purely motivated by my own use, and I only considered `chemicals` because it is the most complete package. We can remove the default value though, since `chemicals` is not a pp dependency.

Reply:
Removed the default value.